drivers/dma/Kconfig.stm32

```diff
@@ -22,9 +22,23 @@ config DMA_STM32_V1
 
 config DMA_STM32_V2
 	bool
-	depends on SOC_SERIES_STM32F0X || SOC_SERIES_STM32F1X || SOC_SERIES_STM32F3X || SOC_SERIES_STM32L0X || SOC_SERIES_STM32L4X
+	depends on SOC_SERIES_STM32F0X || SOC_SERIES_STM32F1X || SOC_SERIES_STM32F3X || SOC_SERIES_STM32L0X || SOC_SERIES_STM32L4X || SOC_SERIES_STM32WBX
 	select USE_STM32_LL_DMA
 	help
-	  Enable DMA support on F0/F1/F3/L0/L4 series SoCs.
+	  Enable DMA support on F0/F1/F3/L0/L4/WB series SoCs.
+
+config DMAMUX_STM32
+	bool
+	depends on DMA_STM32_V2
+	depends on SOC_STM32L4R5XX || SOC_SERIES_STM32WBX
+	help
+	  Enable DMAMUX support on L4R/WB series SoCs.
+
+config DMAMUX_1_NAME
+	string "Device name for DMAMUX Controller 1"
+	depends on DMAMUX_STM32
+	default "DMAMUX_1"
+	help
+	  Device name for DMAMUX Controller 1.
```

Comment:
Is that needed ?
I guess this should not be if using DT macros

Reply:
yes, removed provided through the DT